lib/cluster.js

```diff
@@ -209,7 +221,7 @@ Cluster.prototype.fetchStats = function fetchStats(callback) {
             foundMembers[memberAddr] = true;
             fetchQueue.push(memberAddr, function onStatsFetched(err, stats) {
                 if (err) {
-                    console.error('Error while fetching node stats:', err);
+                    self.connectionErrorCount ++;
```

Comment:
Also, does this ever need to be reset? It is possible that `fetchStats` were to be called multiple times within the context of a single command.